docs/api/version-history.md

```diff
@@ -20,6 +20,16 @@ keywords: "API, Docker, rcli, REST, documentation"
 * The `VirtualSize` field in the `GET /images/{name}/json`, `GET /images/json`,
   and `GET /system/df` responses is now omitted. Use the `Size` field instead,
   which contains the same information.
+* Deprecated: The `is_automated` field in the `GET /images/search` response has
+  been deprecated by Docker Hub's search API, and will always be set to "false"
+  in future. Given that this field is deprecated by an external API, the
+  deprecation will not be versioned, and will apply to any API version.
+* Deprecated: The `is-automated` filter for the `GET /images/search` endpoint.
+  The `is_automated` field has been deprecated by Docker Hub's search API, and
+  will no longer be set in the future. Because of this, the `is-automated` filter
+  will not yield any results when searching for `is-automated=true`.
+  Given that this field is deprecated by an external API, the deprecation will
+  not be versioned, and will apply to any API version.
```

Comment:
I still find the phrasing awkward. It reads like Docker Hub is directly deprecating a part of the Engine API, which doesn't make sense. They are distinct APIs with independent specifications; their only relationship is that one happens to be _implemented_ in terms of the other.

> Deprecated: The `is_automated` field in the `GET /images/search` response has been deprecated and will always be set to false in the future _because Docker Hub is deprecating the `is_automated` field in its search API._ The deprecation _is_ not versioned, and _applies_ to all API versions.
> Deprecated: The `is-automated` filter for the `GET /images/search` endpoint. The `is_automated` field has been deprecated by Docker Hub's search API. Consequently, searching for `is-automated=true` will yield no results. The deprecation is not versioned, and applies to all API versions.

Reply:
Thanks! I updated with your suggestion 👍